packages/lodestar-cli/test/unit/lodestar/account.test.ts

```diff
@@ -0,0 +1,68 @@
+import * as fs from "fs";
+import yargs from "yargs/yargs";
+import { expect } from "chai";
+import * as sinon from "sinon";
+import { assert } from "console";
+
+import {tmpDir} from "../../constants";
+import { account } from "../../../src/cmds/account";
+
+describe("account cli", function() {
+  const spy = sinon.stub(console, 'log');
+  
+  // lodestar account wallet create --name primary --passphraseFile primary.pass --rootDir .tmp
+  it("should create a wallet", async function() {
+    expect(fs.existsSync(`${tmpDir}/wallets/`)).to.be.false;
+    await new Promise(resolve => yargs().default({
+      rootDir: tmpDir,
+      name: "primary",
+      passphraseFile: "primary.pass",
+      // @ts-ignore
+    }).command(account).help().parse(["account", "wallet", "create"], resolve));
+    await new Promise(resolve => setTimeout(resolve, 500));
+    expect(fs.existsSync(`${tmpDir}/wallets/`)).to.be.true;
+  });
+
+  // lodestar account wallet list --rootDir .tmp
+  it("should list existing wallets", async function() {
```

Comment:
This test should assert that the output of the command contains the name of the new wallet